### ISSUE TITLE
"System" function not working

### ISSUE THREAD

**Issue by DiopticTurtle** [omriharel/deej#51]
Good morning,

Firstly, thank you so much for this tool; it is exactly what I was hoping it would be and it has completely changed how I manage sound on my computer in the best way.

On to my issue: it seems that the "System" fader is not changing the system volume; I can verify this by looking at the mixer and as I move the fader the System Sound level does not change. I looked at the previous issues and did not see anything that looked like it was the same issue. I can attest that it did previously function as expected, and I have never touched the config or code since I first implemented Deej.

This is not a major issue, in my opinion, and does not change my opinion of the tool, but it would be nice if I could fix it!

Thank you,

**Comment by omriharel** [comment]
Hi there @DiopticTurtle, thank you for writing. And thanks for the kind words about deej!

As a first thing to try, just to make sure there hasn't been a disconnection of that specific fader at the hardware level - would you be able to assign `system` to another slider in the config, and check if that works? (Or alternatively, binding apps other than `system` to the fader in question and checking that it controls them correctly)

If you've confirmed that the issue is indeed with the `system` binding, and not with the pot itself, I would appreciate any additional information you can provide about your setup: your config file, your precise OS version (including build number), and if possible - a sample output log from deej's debug version (you can find instructions about how to run in debug mode in our [FAQ](https://faq.deej.rocks)).

Thank you!

**Comment by DiopticTurtle** [comment]
Hi @omriharel , I was able to rule out hardware issue, as swapping the bindings of the Master and System faders had the same result (System fader not affecting the volume, Master fader functioning as expected). 

For system info, this is a homebuilt machine and I am running a 64-bit Windows 10 Home, version 10.0.19042 Build 19042. 

I've attached my config yaml (just saved it as a .txt to make the uploader happy) and the text from the Debug .exe after adjusting the Master and System pots. 

[DTconfig.txt](https://github.com/omriharel/deej/files/8098914/DTconfig.txt)
[DTDebugLog.txt](https://github.com/omriharel/deej/files/8098915/DTDebugLog.txt)

Thank you for taking the time and looking into this!

**Comment by omriharel** [comment]
Hey @DiopticTurtle, thanks for the details. I think some of the log contents may have been omitted (I don't see the startup and version info logs that I'm looking for), but I do have another question about your setup:

Have you been using voicemeeter throughout this entire time? I see that its audio sessions and virtual devices are detected by deej, and I'm wondering if we're not experiencing some sort of conflict with downmixing happening inside voicemeeter (which naturally is also closely involved with Windows' audio APIs). Would you be able to try re-opening deej without voicemeeter running, and report back?

The thing that seems odd to me is that deej does see a "system sounds" session, and does seem to be interacting with it - so it's weird that you don't see it reflected in Windows mixer. Perhaps a short screen recording of deej's log view alongside your windows mixer might help make more sense as well.

Thanks!

**Comment by DiopticTurtle** [comment]
Hey @omriharel; that's a good point about VoiceMeter - let me look through the patch and see if I can find anything likely. I'll also re-run the debug and see if I can get the full thing (for some reason I guess it only captured what was on the interface at the time). I'll do a bit of experimenting and see if I can get any changes!

**Comment by DiopticTurtle** [comment]
Hey, apologies for the delay in response - I am no longer experiencing the issue.
I hadn't made many changes to my VoiceMeeter setup although I did just upgrade from Banana to Potato - I had verified that my system was not trying to run system sounds through any VM channel. 
I noticed that System pot was working again after I opened Sound Settings and then the Sound Device Manager; moving the pot affected the level in that application, and then it worked in the Mixer tab after that, functioning as expected. I am marking this as closed.

Thank you for your assistance, and again, apologies in the delay.
